Tests/FoundationInternationalizationTests/CalendarTests.swift

```diff
@@ -1083,6 +1081,48 @@ final class CalendarTests : XCTestCase {
         }
     }
     
+
+    func test_addingDaysAndWeeks() throws {
+        let timeZone = TimeZone(identifier: "America/Los_Angeles")!
+        var c = Calendar(identifier: .gregorian)
+        c.timeZone = timeZone
+
+        let s = Date.ISO8601FormatStyle(timeZone: timeZone)
+        let a = Date(timeIntervalSinceReferenceDate: 731673276) // "2024-03-09T02:34:36-0800", 10:34:36 UTC
+        let d1_w1 = c.date(byAdding: .init(day: 1, weekOfMonth: 1), to: a)!
+        let exp = try Date("2024-03-17T02:34:36-0700", strategy: s)
+        XCTAssertEqual(d1_w1, exp)
+
+        let d8 = c.date(byAdding: .init(day: 8), to: a)!
+        XCTAssertEqual(d8, exp)
+    }
+
+    func test_addingDifferencesRoundtrip() throws {
+        let timeZone = TimeZone(identifier: "America/Los_Angeles")!
+        var c = Calendar(identifier: .gregorian)
+        c.timeZone = timeZone
+
+        let s = Date.ISO8601FormatStyle(timeZone: timeZone)
+        let tests = [
+            // 2024-03-09T02:34:36-0800, 2024-03-17T03:34:36-0700, 10:34:36 UTC
+            (Date(timeIntervalSinceReferenceDate: 731673276), Date(timeIntervalSinceReferenceDate: 732364476)),
+
+            // 2063-03-10T02:27:06-0800, 2063-03-18T03:27:06-0700
+            (Date(timeIntervalSinceReferenceDate: 1962440826), Date(timeIntervalSinceReferenceDate: 1963132026)),
+
+            // 2024-03-08T02:34:36-0800, 2063-03-18T11:27:06-0700
+            (Date(timeIntervalSinceReferenceDate: 731586876.690495), Date(timeIntervalSinceReferenceDate: 1963160826.550588)),
+            
+            // 2024-03-03T02:34:36-0800, 2024-03-11T02:34:36-0700
+            (Date(timeIntervalSinceReferenceDate: 731154876), Date(timeIntervalSinceReferenceDate: 731842476)),
+        ]
+
+        for (start, end) in tests {
```

Comment:
I prefer tests to not loop over samples as it makes it hard to tell which sample caused the failure, especially in CI...but your choice!

Reply:
I absolutely agree with you. I was just being lazy.